pandas/io/tests/test_parsers.py

```diff
@@ -3070,6 +3070,26 @@ def test_whitespace_lines(self):
         df = self.read_csv(StringIO(data))
         tm.assert_almost_equal(df.values, expected)
 
+    @tm.network
```

Comment:
shouldn't this test be the same as the one below (e.g. with the same URL), except for the difference in the compression arg?